src/main/webapp/WEB-INF/academic-configurations/statutetypemanagement/statutetype/update.jsp

```diff
@@ -0,0 +1,222 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags"%>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jstl/fmt" %>
+<spring:url var="datatablesUrl" value="/javaScript/dataTables/media/js/jquery.dataTables.latest.min.js"/>
+<spring:url var="datatablesBootstrapJsUrl" value="/javaScript/dataTables/media/js/jquery.dataTables.bootstrap.min.js"></spring:url>
+<script type="text/javascript" src="${datatablesUrl}"></script>
+<script type="text/javascript" src="${datatablesBootstrapJsUrl}"></script>
+<spring:url var="datatablesCssUrl" value="/CSS/dataTables/dataTables.bootstrap.min.css"/>
+<link rel="stylesheet" href="${datatablesCssUrl}"/>
+<spring:url var="datatablesI18NUrl" value="/javaScript/dataTables/media/i18n/${portal.locale.language}.json"/>
+
+<link rel="stylesheet" type="text/css"
+	href="${pageContext.request.contextPath}/CSS/dataTables/dataTables.bootstrap.min.css"/>
+
+<link href="//cdn.datatables.net/responsive/1.0.4/css/dataTables.responsive.css" rel="stylesheet"/>
+<script src="//cdn.datatables.net/responsive/1.0.4/js/dataTables.responsive.js"></script>
+<link href="//cdn.datatables.net/tabletools/2.2.3/css/dataTables.tableTools.css" rel="stylesheet"/>
+<script src="//cdn.datatables.net/tabletools/2.2.3/js/dataTables.tableTools.min.js"></script>
+<link href="//cdnjs.cloudflare.com/ajax/libs/select2/4.0.0-rc.1/css/select2.min.css" rel="stylesheet" />
```

Comment:
Please don't add external resource links, specially to libraries in Release Candidate :)